django/core/validators.py

```diff
@@ -397,8 +398,12 @@ class StepValueValidator(BaseValidator):
     message = _("Ensure this value is a multiple of step size %(limit_value)s.")
     code = "step_size"
 
+    def __init__(self, limit_value, offset=None, message=None):
```

Comment:
This should be documented in `StepValueValidator` [docs](https://docs.djangoproject.com/en/dev/ref/validators/#stepvaluevalidator) in the `StepValueValidator` signature, content, and `versionchanged` annotation.

Reply:
We should also mention it in the `forms.IntegerField`, `forms.FloatField`, and `forms.DecimalField` docs where `StepValueValidator` is mentioned.